sources/packages/backend/workflow/test/2021-2022/parttime-assessment/awards-amounts/parttime-assessment-2021-2022-awards-amount-BCAG.e2e-spec.ts

```diff
@@ -127,4 +127,31 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-BC
     expect(calculatedAssessment.variables.awardEligibilityBCAG).toBe(false);
     expect(calculatedAssessment.variables.provincialAwardNetBCAGAmount).toBe(0);
   });
+
+  it("Should determine provincialAwardNetBCAGAmount when awardEligibilityBCAG is true and provincialAwardPartTimeBCAGAmount is null", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataCRAReportedIncome = 20001;
+    assessmentConsolidatedData.studentDataRelationshipStatus = "married";
+    assessmentConsolidatedData.studentDataIsYourSpouseACanadianCitizen =
+      YesNoOptions.Yes;
+    assessmentConsolidatedData.partner1CRAReportedIncome = 22999;
+    // Public institution
+    assessmentConsolidatedData.institutionType = InstitutionTypes.BCPublic;
+    assessmentConsolidatedData.programYearTotalPartTimeBCAG = null;
```

Comment:
can we set the value to undefined? (same in other places)

Reply:
i believe creating the value as null or undefined both are wrong and the right way would be without populating the Programyeartotal{offeringIntensity}{AwardType} in the createFakeConsolidatedPartTimeData. Let have a chat and we can close this.

Reply:
@guru-aot  _**without populating the Programyeartotal{offeringIntensity}{AwardType}**_ -  is same as setting undefined which is why i mentioned it.